.travis.yml

```diff
@@ -4,19 +4,27 @@ language: ruby
 cache:
   bundler: true
 gemfile:
+- Gemfile.rails-6-0
 - Gemfile.rails-5-2
 - Gemfile.rails-5-1
 - Gemfile.rails-5-0
 - Gemfile.rails-master
 rvm:
 - 2.4.1
+- 2.5.3
+- 2.6.2
```

Comment:
Thanks for extending the matrix!

Could you amend the file to use these?

- 2.4.6
- 2.5.5
- 2.6.3

Reply:
Yup! Done in 67f12ef